frontend/src/app/modules/distribution-chart/distribution-chart.component.html

```diff
@@ -0,0 +1,4 @@
+<osm-menu></osm-menu>
+<p>
+  {{ 'frontend.de.iteratec.osm.aggregation.description' | translate }}
```

Comment:
It should be used the property for "distributionChart" instead of "aggregation".